### DISCUSSION TITLE
Is is possible to load accounts from a wallet's seed phrase?

### DISCUSSION THREAD

**Discussion by RealJohnnyTime** [foundry-rs/starknet-foundry#1566] [General]
Hey everyone, I'm curious if it's possible to load my accounts into the `.toml` file using the seed phrase I created through a wallet like Braavos or Argent.

Is it possible with Starknet Foundry?
Thanks!

**Comment by Arcticae** [comment]
Unfortunately we don't support loading account from seed phrases, we keep our integration limited in that matter
See more info on how we support importing accounts [here](https://foundry-rs.github.io/starknet-foundry/starknet/account.html)
